twisted/conch/ssh/transport.py

```diff
@@ -1297,6 +1371,8 @@ def ssh_KEX_DH_GEX_REQUEST_OLD(self, packet):
         # another cue to decide what kind of message the peer sent us.
```

Comment:
this comment got out of sync with the code. You might want to consider repharsing the comment or moving this code into a separate method... or refactor it somehow :)